### PR TITLE
Stethoscope fixes

### PR DESCRIPTION
**Pull request by QualityVan**
:cl: QualityVan
fix: Improved stethoscopes
/:cl:

Changes stethoscopes to work on heart and lung health/presence rather than just oxloss, and lets stethoscopes detect facehugger eggs.

### CODE REVIEW

**Comment by coiax** [comment]
Balance. Open after the freeze.

**Comment by QualityVan** [comment]
I asked whether this would count as a fix in coderbus and was told yes, although I don't have the chat log handy. Specifically, this fixes seeing a pulse and respiration for someone who doesn't have a heart and lungs. Would cutting the aylium bit help?

**Comment by MrStonedOne** [comment]
ya, the xeno bit puts it over in the feature direction.

**Comment by PKPenguin321** [comment]
Big :-1: this removes the ability to target the groin with these how the fuck are you supposed to do Johnson inspections

**Comment by QualityVan** [comment]
You'll want to code microscopes for that, PKP.

**Comment by PKPenguin321** [comment]
Oh shit

File: code/modules/clothing/neck/ties.dm [hunk 1]
```diff
@@ -38,34 +38,28 @@
...
-							sound = "pulse and respiration"
-						if("eyes","mouth")
-							sound_strength = "cannot hear"
-							sound = "anything"
-						else
-							sound_strength = "hear a weak"
-
-				user.visible_message("[user] places [src] against [M]'s [body_part] and listens attentively.", "You place [src] against [their] [body_part]. You [sound_strength] [sound].")
+			if(body_part != "chest")
+				to_chat(user, "<span class='warning'>Stethoscopes only work on the chest!</span>")
```
Comment (octareenroon91) [inline code/modules/clothing/neck/ties.dm]: Actually this is inappropriate: you can pick up a pulse in the extremities. It's key to measuring blood pressure.

Comment (QualityVan) [inline code/modules/clothing/neck/ties.dm]: Suggestions on how heart failure should interact with limb examinations?

Comment (octareenroon91) [inline code/modules/clothing/neck/ties.dm]: You get just a pulse instead of the rushing air and blood sounds that you'd hear in the chest, I guess.

**Comment by QualityVan** [comment]
Groin targeting has been reenabled, you may rejoice.

**Pull request merged**